technical-reference/reference-technical-articles/xvs-bridge.md

```diff
@@ -212,6 +214,7 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
    |--------------------|-------------------|----------------------|------------------|---------------------|
    | BNB <--> Ethereum  | $100,000          | $102,000             | $1,000,000       | $1,020,000          |
    | BNB <--> opBNB     | $10,000           | $10,200              | $50,000          | $51,000             |
+   | BNB <--> Arbitrum  | $20,000           | $20,400              | $100,000         | $102,000            |
    | Ethereum <--> opBNB| $10,000           | $10,200              | $50,000          | $51,000             |
 
```

Comment:
I would add the limits between Arbitrum and Ethereum, and between Arbitrum and opBNB

Reply:
Done https://github.com/VenusProtocol/venus-protocol-documentation/pull/204/commits/420cfaa53d80480f1c941e163cef6809a41f9053